lib/media/example_based_criteria.js

```diff
@@ -20,11 +20,12 @@ shaka.media.ExampleBasedCriteria = class {
    * @param {shaka.config.CodecSwitchingStrategy} codecSwitchingStrategy
    * @param {shaka.media.AdaptationSetCriteria.Factory
    *        } adaptationSetCriteriaFactory
+   * @param {string} role
    */
-  constructor(example, codecSwitchingStrategy, adaptationSetCriteriaFactory) {
-    // We can't know if role and video label are really important, so we don't
-    // use role and video label for this.
-    const role = '';
```

Comment:
Why not get the audio role here instead of passing it as a parameter?

Reply:
Yes, extracting it inside ExampleBasedCriteria is much better. Thanks for catching that. I don't know why I decided to pass it through player.js. I will update it. One sec.